### ISSUE TITLE
[css-overflow-5] Limit scrolling to the associated scroll container when activating a marker

### ISSUE THREAD

**Issue by flackr** [w3c/csswg-drafts#11138]
Currently, when a scroll marker is activated, the [activation behavior](https://drafts.csswg.org/css-overflow-5/#scroll-marker-activation) [scrolls the scroll target into view](https://drafts.csswg.org/cssom-view-1/#scroll-a-target-into-view). This behavior right now will scroll every scrolling box up to the root to the target.

I think that it is quite common to expect that the scroll stops at the managed scroll container. E.g. @bakura10 from twitter mentioned this expectation in https://github.com/w3c/csswg-drafts/issues/9452#issuecomment-1756829797. In #9452 I proposed adding this option to the scroll into view api.

The question here is, should we restrict the scroll into view for activation to stop at the [scroll marker group container](https://drafts.csswg.org/css-overflow-5/#scroll-marker-group-container)? This would mean that we would not ensure that the group container is currently scrolled into view in all of it's ancestor scrolling containers.

If we don't restrict this, you could end up in a degenerate situation when the scroll markers are above the targeted [scroll marker group container](https://drafts.csswg.org/css-overflow-5/#scroll-marker-group-container) as scrolling the ancestors of the group container to position the group container into view would naturally scroll the markers out of view.

I propose we do this.

**Comment by tabatkins** [comment]
Yes, I think we absolutely need to add the "stop at this element" option behavior to the scrolling APIs, and auto-invoke it for marker activation to stop at the scroll marker group container.

So, proposals:

1. Accept the proposal in #1522, defining a new `Element.scrollingParent` accessor that returns the nearest scrollable ancestor of an element.
2. Then, accept the proposal in #9452, defining a new `container` option for the `scrollIntoView()` method, taking an optional Element to stop at. (The previous one makes it easy to just stop at the nearest scroller.)
3. Then, accept the proposal here, to make scroll-marker scrolling automatically stop at the scroll marker group container.

**Comment by flackr** [comment]
Spec edits are landed in #11673.
